cluster-up/README.md

```diff
@@ -1,6 +1,6 @@
 # How to use cluster-up
 
-This directory provides a wrapper around gocli. It can be vendored into other
+This directory provides a wrapper around gocli. It can be vended into other
```

Comment:
I think `vendored` is ok here - the rest look good to me.

Reply:
updated, thanks.